### ISSUE TITLE
[Suggestion] Rooty Soil doesn't convert to dirt.

### ISSUE THREAD

**Issue by UnlimatedStone9** [Gaelmare/dynamictreestfc#21]
Per Discord conversation I'm not a fan of the soil dirt system, that's not a fault of yours, it's just a personal preference.
Could a toggle to prevent rooty soil converting into dirt at all be implemented? This means once soil becomes rooty, it is from then onwards rooty soil. This means I could change how the block works in modpacks to instead drop roots and such, and allow reproducing dirts in other ways.
I'm cruel. This config would be amazing.

TLDR: ``doesRootySoilDirtify=false``

**Comment by UnlimatedStone9** [comment]
- Be me.
- Think of this idea.
- Come to GitHub to see I already came up with this idea...

**Comment by Gaelmare** [comment]
See DT podzol settings. Not sure how this works for DTTFC.
